x-pack/plugins/ingest_manager/server/plugin.ts

```diff
@@ -52,12 +52,14 @@ import {
   AgentService,
 } from './services';
 import { getAgentStatusById } from './services/agents';
+import { CloudSetup } from '../../cloud/server';
 
 export interface IngestManagerSetupDeps {
   licensing: LicensingPluginSetup;
   security?: SecurityPluginSetup;
   features?: FeaturesPluginSetup;
   encryptedSavedObjects: EncryptedSavedObjectsPluginSetup;
+  cloud?: CloudSetup;
```

Comment:
Since we're adding this to our dependencies, I believe we'd need to add to `kibana.json` as an optional dependency https://github.com/elastic/kibana/blob/5a99642cfbebc0611b15666244aa5c91ec2935f9/x-pack/plugins/ingest_manager/kibana.json#L8